README.md

```diff
@@ -569,6 +569,18 @@ chatSession.onAutoDisconnection(event => {
 ```
 `onParticipantIdle`, `onParticipantReturned`, and `onAutoDisconnection` are related to [set up chat timeouts for chat participants](https://docs.aws.amazon.com/connect/latest/adminguide/setup-chat-timeouts.html).
 
+#### `chatSession.onConnectionLost()`
+```js
+chatSession.onConnectionLost(event => {
+  const { chatDetails, data } = event;
+  switch (data.ContentType) {
```

Comment:
I believe `data` here will always have the value `{}`. So `data.ContentType` will be `undefined`.